internal/pkg/utils/error.go

```diff
@@ -11,50 +12,66 @@ import (
 
 type multiError = multierror.Error
 
-type Error struct {
+type MultiError struct {
 	*multiError
 	lock *sync.Mutex
 }
 
-type RawError struct {
+type NestedError struct {
```

Comment:
Vnorena chyba, vysledok `PrefixError`, napr:
```
bad local state: <<< msg
- a  <<< MultiError
- b
```

Reply:
To tu bolo aj predtym, ale vramci `PrefixError` sa to hned previedlo na `string`, ... lenze potom uz neviem zistit typy povodnych chyb, ktore tam boli (co potrebujem v dalsich PR). 
Preto sa to ulozi ako `NestedError`, a az pri vypisovani sa to prevedie na string.